### ISSUE TITLE
Accessibility issue - portlet List of spaces

### ISSUE THREAD

**Issue by margondicco** [Meeds-io/meeds#2414]
Error 1 : Elements with an ARIA [role] that require children to contain a specific [role] are missing some or all of those required children.

<img width="704" alt="image" src="https://github.com/user-attachments/assets/4f2b5f83-d7ca-425e-9f17-9caef5ca7899">

Error 2 : Uses ARIA roles on incompatible elements
<img width="700" alt="image" src="https://github.com/user-attachments/assets/0c80e819-5854-4caa-8511-af46ac85f0a5">
